quic/s2n-quic-core/src/transport/parameters/mod.rs

```diff
@@ -215,7 +215,7 @@ where
 }
 
 #[derive(Copy, Clone, Debug, PartialEq)]
-pub struct ValidationError(&'static str);
+pub struct ValidationError(pub &'static str);
```

Comment:
I'm not sure we want to make this public... Maybe we implement `AsRef<str>` instead?

Reply:
Not sure I'm following, I made it `pub` so I could initialize a `ValidationError` from outside the transport parameters module. I suppose I could just have a `const INVALID_MIN_RTT: ValidationError` but it seems a little weird to have the error message be in transport parameters, when its not actually a transport parameter. Or I could add a `pub fn new` to `ValidationError`?